cloudbuild/local-cloudbuild.yaml

```diff
@@ -0,0 +1,94 @@
+steps:
```

Comment:
not clear what this file is useful for--for running `gcloud build` from your workstation?

Reply:
Using the same file locally will pull from github branch, which is not conducive for development.

I agree that it's sad we are repeating ourselves here - maybe we can parameterize `cloudbuild.yaml` to not pull from github, but I wanted to consider that out of scope here.

Reply:
Removed.